### DISCUSSION TITLE
How to open a link?

### DISCUSSION THREAD

**Discussion by MasterRO94** [NativePHP/laravel#173] [Q&A] [answered]
Is there a way to redirect users away from the app to a browser, e.g. open an external link?
Now it opens inside app

**Comment by osc2nuke** [comment]
I tested it out, but links appear for now to only work via the Applications Menu & Menu Bar.
But i agree with the cumberstone.
Perhaps an attribute like:
`<a href="https:/nativephp.com" target="_native">Visit NativePHP</a>`

**Reply by MasterRO94** [reply]
Yeah, need some kind of API to trigger Electron's `shell.openExteranl(url)`

**Comment by MasterRO94** [comment, marked as answer]
Found a solution.

NativePHP has `Shell` helper class. So to open an external url you can call `Shell::openExternal($url)`

Full example 
```blade
<a
  class="block w-20 h-6 bg-blue-500 rounded-lg text-center text-white hover:bg-blue-400"
  href="#!"
  wire:click.prevent="openExternal('{{ $meeting->url }}')"
 >
   Join
</a>
```

```php
use Native\Laravel\Facades\Shell;

// 

public function openExternal(string $url): void
{
    Shell::openExternal($url);
}
```

**Comment by osc2nuke** [comment]
Very good!
